test/e2e/app-dir/scss/basic-module-implementation-embedded-sass/basic-module-implementation-sass-embedded.test.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-env jest */
+
+import { nextTestSetup } from 'e2e-utils'
+import { colorToRgb } from 'next-test-utils'
+
+describe('Basic Module Prepend Data Support', () => {
+  const { next } = nextTestSetup({
```

Comment:
In order to be sufficient I think we have to duplicate the entire Sass test suite run to also use sass-embedded. Should be pretty straightforward now that these are in `test/e2e/app-dir/scss`, could just add a `describe.each` around the setup to swap the implementation.

Reply:
Added tests, see https://github.com/vercel/next.js/pull/64577/commits/652602d634663c78b2699511f6a8a83ca5e049c1